packages/logging/src/log.js

```diff
@@ -592,25 +593,17 @@ Log.prototype.write = function(entry, options, callback) {
     options = {};
   }
 
-  var protoOpts = {
-    service: 'LoggingServiceV2',
-    method: 'writeLogEntries'
-  };
-
-  var reqOpts = extend({
-    logName: this.formattedName_
-  }, options);
-
-  var entries = arrify(entry);
-
-  this.decorateEntries_(entries, function(err, decoratedEntries) {
+  this.decorateEntries_(arrify(entry), function(err, decoratedEntries) {
     // Ignore errors (the API will speak up if it has an issue).
 
-    reqOpts.entries = decoratedEntries;
+    var reqOpts = extend({
+      logName: self.formattedName_,
+      entries: decoratedEntries
+    }, options);
 
-    self.request(protoOpts, reqOpts, function(err, resp) {
-      callback(err, resp);
-    });
+    delete reqOpts.gaxOptions;
+
+    self.api.Logging.writeLogEntries(reqOpts, options.gaxOptions, callback);
```

Comment:
@jmuk this is printing output, from the system tests:

```
no such bundleid: ["projects/nth-circlet-705/logs/syslog",{"type":"gce_instance","labels":{"zone":"global","instance_id":"3"}},null]
```

I'm not sure if I'm doing something wrong, but it's not coming through in the error handler.